doc/rbd/rbd-openstack.rst

```diff
@@ -232,15 +233,28 @@ Edit ``/etc/glance/glance-api.conf`` and add under the ``[glance_store]`` sectio
 
 For more information about the configuration options available in Glance please refer to the OpenStack Configuration Reference: http://docs.openstack.org/.
 
-Any OpenStack version
-~~~~~~~~~~~~~~~~~~~~~
+Enable copy-on-write cloning of images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Note that this exposes the back end location via Glance's API, so the endpoint
+with this option enabled should not be publicly accessible.
+
+Before Mitaka
+^^^^^^^^^^^^^
 
 If you want to enable copy-on-write cloning of images, also add under the ``[DEFAULT]`` section::
 
     show_image_direct_url = True
 
-Note that this exposes the back end location via Glance's API, so the endpoint
-with this option enabled should not be publicly accessible.
+As of Mitaka
+^^^^^^^^^^^^
+
+To enable image locations and take advantage of copy-on-write cloning for images, add under the ``[DEFAULT]`` section::
+
+    show_multiple_locations = True
+
+Disable cache management (any OpenStack version)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I always took these are separate issues since the caching would apply on image GETs, but if direct url was enabled there wouldn't be a GET issued against the image.  I could be mistaken since it's been a couple years since I used OpenStack.

However, looking at the code, I did notice that this new param is already deprecated and the ```show_image_direct_url``` parameter still exists and functions [1][2], so I guess the advice should be to keep show_image_direct_url and for Mikata only, additionally set show_multiple_locations.

[1] https://github.com/openstack/glance/blob/master/glance/common/config.py#L375
[2] https://github.com/openstack/glance/blob/master/glance/api/v2/images.py#L779